### DISCUSSION TITLE
Trace Route command in OpenThread CLI

### DISCUSSION THREAD

**Discussion by sohaibbinaltaf** [openthread/openthread#8603] [Q&A] [answered]
Hello, 

I want to know if there is any command which can give details of the route from source node to destination, and if we can change the route by ourselves what commands can be used.  

I have developed a thread network using nRF52840 dongle and Raspberry Pi 4B. I have checked from the command line of the RPi, the transmit power is 0dBm. Can you please confirm if we can increase this transmit power from the command line?

**Comment by jwhui** [comment, marked as answer]
> I want to know if there is any command which can give details of the route from source node to destination, and if we can change the route by ourselves what commands can be used.

For routing within the Thread mesh topology, see [`route table`](https://github.com/openthread/openthread/tree/main/src/cli#router-table). OpenThread does not currently provide a way to manually update route table.

> I have developed a thread network using nRF52840 dongle and Raspberry Pi 4B. I have checked from the command line of the RPi, the transmit power is 0dBm. Can you please confirm if we can increase this transmit power from the command line?

See [`txpower`](https://github.com/openthread/openthread/tree/main/src/cli#txpower-txpower).
